src/internal/connector/exchange/service_iterators.go

```diff
@@ -162,50 +163,42 @@ func FetchEventIDsFromCalendar(
 	gs graph.Service,
 	user, calendarID string,
 ) ([]string, error) {
-	ids := []string{}
+	var (
+		errs *multierror.Error
+		ids  []string
+	)
 
-	response, err := gs.Client().
+	query := gs.Client().
```

Comment:
Without the `Get`,  these are called request builders. I have been using the name builder for short. `query` doesn't quite contain what the variable is. Especially when it is updated on line 201.